docs/alma/alma.rst

```diff
@@ -178,6 +178,25 @@ or if you wanted all projects by a given PI:
 
     >>> Alma.query(payload=dict(pi_name='Ginsburg, Adam'))
 
+Querying For Proprietary Data
+-----------------------------
+
+To query for proprietary data (non-public data), you need to first authenticate using the ``login`` method,
+and then set ``public_data=False`` in your query:
+
+.. doctest-skip::
+
+    >>> from astroquery.alma import Alma
+    >>> alma = Alma()
+    >>> # First login to access proprietary data
+    >>> alma.login("username")  # Will prompt for password
+    >>> # Now query including proprietary data
+    >>> proprietary_data = alma.query(payload=dict(project_code='2017.1.01355.L', public_data=False))
+    >>> # Or restrict to only proprietary data
+    >>> only_proprietary = alma.query_region('W51', radius=25*u.arcmin, public_data=False)
```

Comment:
noting that the `query_region` function takes `public` not `public_data` as the keyword.  The issue is correct but the doc change isn't.

Reply:
thanks for catching that, it was indeed the problem.  It comes from `Alma.help()` showing `public_data`, not `public`.